src/main.rs

```diff
@@ -136,7 +136,14 @@ fn run() -> Result<(), failure::Error> {
                 .about(&*format!(
                     "{} Setup wrangler with your Cloudflare account",
                     emoji::SLEUTH
-                )),
+                ))
+                .arg(
+                    Arg::with_name("token")
+                        .help("use an API token for authentication. API tokens are still in beta")
```

Comment:
nit: use an API token instead of your email and global API key for authentication. maybe add a link to where they can set up api tokens in dashboard in help message

Reply:
that link depends on what the user's account is... so I'm not the biggest fan of providing it. How about we just tell users that they can issue a token via the dashboard?